README.md

```diff
@@ -50,16 +50,36 @@ Use for:
 
 ## Getting Started with Local Tooling
 
+### Download Skaffold binary 
+
+-  To download the latest Linux build, run:
+    -  `curl -Lo skaffold https://storage.googleapis.com/skaffold/releases/latest/skaffold-linux-amd64 && chmod +x skaffold && sudo mv skaffold /usr/local/bin`
+-  To download the latest OSX build, run:
+    -  `curl -Lo skaffold https://storage.googleapis.com/skaffold/releases/latest/skaffold-darwin-amd64 && chmod +x skaffold && sudo mv skaffold /usr/local/bin`
+
+### Build from source TL;DR
+
+0. Install Go and [dep](https://golang.github.io/dep/)
+
+1. Get the source 
+
+        $ go get github.com/GoogleCloudPlatform/skaffold
+        package github.com/GoogleCloudPlatform/skaffold: no Go files in /path/to/go/workspace/src/github.com/GoogleCloudPlatform/skaffold
+
+1. Update and build all dependencies
```

Comment:
This is actually unnecessary, skaffold vendors all of its dependencies so `make install` should work out of the box